src/mesh/interpolation.cxx

```diff
@@ -68,8 +68,11 @@ const Field3D interp_to(const Field3D &var, CELL_LOC loc, REGION region) {
     if (region != RGN_NOBNDRY) {
       // result is requested in some boundary region(s)
       result = var; // NOTE: This is just for boundaries. FIX!
+      result.allocate();
+    } else {
+      result.allocate();
+      invalidateGuards(result);
```

Comment:
I don't think this `invalidateGuards` call is needed as it should be done in `allocate` if `result` doesn't already have a data block.